### PR TITLE
Allow to read state by store on action creator callback

### PR DESCRIPTION
**Pull request by emmenko**
@gaearon Is that what you meant? It looked too much easier to implement... O.O

### CODE REVIEW

**Comment by gaearon** [comment]
Yeah, this seems right! Let's keep this out of the examples though. It's fine in README, I might have a proper example later. It's OK that the method is unused for now.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 redux
 =========================
 
-An experiment in fully hot-reloadable Flux.
```
Comment (gaearon) [inline README.md]: Whitespace is significant in markdown files. Otherwise it'll remove the line breaks where I want them.

Comment (emmenko) [inline README.md]: Oh sorry, I have `"trim_trailing_white_space_on_save": true` on my sublime :)
Will try to revert it...

**Comment by emmenko** [comment]
Regarding the example, sure, I was just making sure it was working :)

I'll revert that as well

**Comment by emmenko** [comment]
@gaearon done :)

**Comment by gaearon** [comment]
Good, can you please squash into a single commit?

**Comment by emmenko** [comment]
Here we go :)

**Comment by gaearon** [comment]
:+1:

**Comment by gaearon** [comment]
Out in 0.7.0.

**Pull request merged**